### PR TITLE
482 frontend user signup confirm details

### PR DESCRIPTION
**Pull request by asun555**

### CODE REVIEW

**Comment by asun555** [comment]
https://github.com/UoaWDCC/uasc-web/assets/126216753/9f78b9ae-56bc-4367-8a5d-05346b344966

**Review by bcho892 — APPROVED** [review]
the "bert"

**Comment by bcho892** [comment]
also the thing incorrectly says skiier tho

File: client/src/components/composite/SignUpForm/Sections/ConfirmDetailsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            <MyText text={emergency_contact} />
+          ) : (
+            <MyText text="N/A" />
+          )}{" "}
+          <SectionSeparator />
+          <Title text="Additional Information" />
+          <Field text="Skier/Snowboarder?" />
+          <MyText
+            text={
+              does_ski && does_snowboarding
```
Comment (bcho892) [inline client/src/components/composite/SignUpForm/Sections/ConfirmDetailsSection.tsx:106]: this logic isn't exactly correct

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: client/src/components/composite/SignUpForm/Sections/ConfirmDetailsSection.tsx (hunk 1 above)
Comment (bcho892) [reply client/src/components/composite/SignUpForm/Sections/ConfirmDetailsSection.tsx:106]: you need to include the "new option"

**Review by bcho892 — APPROVED** [review]
bert

**Pull request merged**